service/lib/dinstaller/dbus/manager.rb

```diff
@@ -90,6 +101,32 @@ def progress
           prg.current_minor_step
         ].freeze
       end
+
+    private
+
+      # @return [Logger]
+      attr_reader :logger
+
+      # @return [DInstaller::Manager]
+      attr_reader :backend
+
+      # Adds callback to be called when the status changes
+      #
+      # The callback will emit a signal
+      def add_status_callback
```

Comment:
well, in general I am not sure about names. I would use something like `register_status_callback` as this method sounds like you add callback to this method and not registering if to backend.